peer/info.go

```diff
@@ -0,0 +1,80 @@
+package peer
+
+import (
+	"fmt"
+	"strings"
+
+	ma "github.com/multiformats/go-multiaddr"
+)
+
+// PeerInfo is a small struct used to pass around a peer with
+// a set of addresses (and later, keys?).
+type Info struct {
```

Comment:
Please don't rename the type, it should still be called `PeerInfo`.

Reply:
Except it now stutters with the package name or always did

Reply:
`peer.PeerInfo` is not idiomatic. It used to be `pstore.PeerInfo`, that's why the `Peer` qualifier was required. This type is now in the place where it belongs, and the qualifier becomes redundant.

However, `peer.Info` _is_ vague and unhelpful. Possible alternatives:

* `peer.DialInfo`
* `peer.AddrInfo`
* `peer.AddressSlip`

Reply:
`peer.AddrInfo` seems the most pertinent; it _is_ addr information about a peer.